.github/build-mandrel-images.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+PREFIX_NAME=quay.io/quarkus/ubi-quarkus-mandrel
+IMAGE=quarkus-mandrel.yaml
+VERSIONS=('20.1.0.0.Alpha1-java11')
+BUILD_ENGINE=podman
+
+virtualenv --python=python3 ~/cekit
+source ~/cekit/bin/activate
+
+for version in "${VERSIONS[@]}"
+do
+    echo "Generating ${PREFIX_NAME}:${version}"
+    cekit --descriptor ${IMAGE} build \
+        --overrides "{'version': '${version}', 'modules': {'install': [{'name':'mandrel', 'version': '${version}'}]}}" \
+        ${BUILD_ENGINE} --tag="${PREFIX_NAME}:${version}"
+done
+
+docker image prune -f
```

Comment:
Shouldn't this be `${BUILD_ENGINE}` instead of `docker`?

Reply:
Good catch @jerboaa the same "issue" appears in https://github.com/quarkusio/quarkus-images/blob/master/.github/build-native-images.sh (where I copied this from).

Reply:
This is now resolved (for mandrel images only though)